src/applications/disability-benefits/all-claims/components/Autocomplete.jsx

```diff
@@ -136,16 +136,19 @@ const Autocomplete = ({
       />
       {results.length > 0 && (
         <ul
-          aria-hidden="true"
```

Comment:
Did some testing on my iphone - Safari/VoiceOver and I coudn't select anything from the dropdown, so hopefully removing this aria-hidden will allow that interaction again on mobile.